python/ray/tests/test_runtime_env.py

```diff
@@ -330,13 +318,13 @@ def f():
     """
     Test task raises an exception.
     """
-    with pytest.raises(RuntimeEnvSetupError):
+    with pytest.raises(ray.exceptions.LocalRayletDiedError):
```

Comment:
Can you check there's the error message that says the reason is due to agent failure?

Reply:
I think I can remove this test. In this test(`test_runtime_env_broken`), we injected a env to make agent failed. Agent failure will lead to raylet failure. Raylet failure will lead to driver failure(raylet client failed). We can't catch any error message.